### PR TITLE
code sanitization cherry-picked from upstream push

### PR DESCRIPTION
**Pull request by jjsjann123**
revert some unintended changes.

### CODE REVIEW

File: test/cpp/jit/test_gpu.cpp [hunk 1]
```diff
@@ -19072,6 +19096,9 @@ TEST(NVFuserTest, PersistentBufferProjection_CUDA) {
 }
 
 TEST(NVFuserTest, FusionIssue1223_CUDA) {
+  if (at::cuda::getDeviceProperties(0)->major < 6) {
```
Comment (naoyam) [inline test/cpp/jit/test_gpu.cpp]: Why are this test and the others skipped?

Comment (csarofeen) [reply test/cpp/jit/test_gpu.cpp]: We should be supporting only Volta+ but I guess these tests empirically failed on the maxwell in the upstream CI

**Review by csarofeen — APPROVED** [review]
Seems fine to me, but do we need to have someone blanket disable C++ tests pre volta?

**Comment by jjsjann123** [comment]
One more thing that I just found from upstream CI.
We are now using `__nanosleep` which is only supported in sm >= 70. Would need to guard those at least. But one more reason to disable pre-volta for cpp tests.

**Comment by csarofeen** [comment]
Yeah, I put nano sleep in for the semaphore check on grid comms so we wouldn't constantly poll the data.

**Comment by jjsjann123** [comment]
Double checked and confirmed cpp tests passing on pascal and volta cards.

**Pull request merged**